VogtObserver2/Trader.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver2
+{
+    class Trader : IObserver
+    {
+        private StockGrabber _stockgrabber = new StockGrabber();
+        private StockGrabber _stockgrabber2 = new StockGrabber();
+        private StockGrabber _stockgrabber3 = new StockGrabber();
+
+        private string _name;
+
+        Random rand = new Random();
```

Comment:
this should be private, make it private and call it `_rand` for consistency (all private variables should start with underscore in C#)